app/models/publication.rb

```diff
@@ -312,6 +312,8 @@ def add_any_pubmed_data_to_hash
 
     pub_hash[:mesh_headings] = pubmed_hash[:mesh_headings] unless pubmed_hash[:mesh_headings].blank?
     pub_hash[:abstract] = pubmed_hash[:abstract] unless pubmed_hash[:abstract].blank?
+    pmc_id = pubmed_hash[:identifier].detect {|id| id[:type] == 'pmc'} if pubmed_hash[:identifier]
+    pub_hash[:identifier] << pmc_id if pmc_id
```

Comment:
ahh, yes, it was blowing up a test when i didn't check for its existence but that is probably because the test data setup was not complete ... let me check again